CI/integration_tests/calculators/test_green_kubo_self_diffusion_coefficients.py

```diff
@@ -99,8 +99,8 @@ def test_calculator(tmp_path):
     time_should_be = time_step * np.arange(0, vacf_range) * units.time
     thermal_vel_SI = np.sqrt(3 * kT / mass) * units.length / units.time
     relaxation_time_SI = relaxation_time * units.time
-    vacf_should_be = thermal_vel_SI ** 2 * np.exp(-time_should_be / relaxation_time_SI)
-    diff_coeff_should_be = diff_coeff * units.length ** 2 / units.time
+    vacf_should_be = thermal_vel_SI**2 * np.exp(-time_should_be / relaxation_time_SI)
```

Comment:
Do you know the rational behind this change?

Reply:
Black changed its formating style going to the stable release with regards to `2**2` vs `2 ** 2`